src/foundry/client/apps/sidebar/apps/keybindings-config.d.mts

```diff
@@ -65,7 +63,7 @@ declare global {
      * Handle Control clicks
```

Comment:
_onResetDefaults should be typed as returning `void`, based on my reading of the code

Reply:
Guess I wasn't sure what Dialog.confirm eventually returns. But it should be `Promise<void>` then right? (as it's an async function)

Reply:
Yeah `Promise<void>`